### ISSUE TITLE
Dropdown prevents default behavior of Alt+Down when onKeyDown overridden

### ISSUE THREAD

**Issue by AkshanshSrivastava03** [OfficeDev/office-ui-fabric-react#4339]
### Bug Report

- __Package version(s)__: office-ui-fabric-react@5.55.1004

#### Describe the issue:

In drop down, one can pass onKeyDown as a prop to override the default key down functionality.
This has an issue. In the function, private _onDropdownKeyUp(ev: React.KeyboardEvent<HTMLDivElement>), we first send the props the event, and then check if(event.preventDefault) and if that is true we return and stop executing the default behavior (switch case as per key code).

This logic looks flawed to me as event.preventDefault is a function and checking its existence will return true only. 

#### Actual behavior:
When one passes onKeyDown function as a prop to the drop down control, drop down stops showing its drop down items on press of Alt + Down arrow key / on press on Enter key.

Basically all behavior specified in the on key down function will stop working as the function returned

#### Expected behavior:

Alt+Down arrow key should always open/close the drop down.

### If applicable, please provide a codepen repro:

https://codepen.io/AkshanshSrivastava03/pen/wmdEVJ
